robotoff/slack.py

```diff
@@ -96,7 +106,57 @@ def _slack_message_block(
     return [block]
 
 
-class SlackNotifier(SlackNotifierInterface):
+class MultiNotifier(NotifierInterface):
+    """Aggregate multiple notifiers in one instance
+
+    See NotifierInterface for methods documentation
+
+    :param notifiers: the notifiers to dispatch to
+    """
+
+    def __init__(self, notifiers: List[NotifierInterface]):
+        self.notifiers: List[NotifierInterface] = notifiers
+
+    def _dispatch(self, function_name: str, *args, **kwargs):
+        """dispatch call to function_name to all notifiers"""
+        for notifier in self.notifiers:
+            fn = getattr(notifier, function_name)
+            fn(*args, **kwargs)
+
+    def notify_image_flag(
+        self, predictions: List[Prediction], source_image: str, barcode: str
+    ):
+        self._dispatch("notify_image_flag", predictions, source_image, barcode)
+
+    def notify_automatic_processing(self, insight: ProductInsight):
+        self._dispatch("notify_automatic_processing", insight)
+
+    def send_logo_notification(
+        self, logo: LogoAnnotation, probs: Dict[LogoLabelType, float]
+    ):
+        self._dispatch("send_logo_notification", logo, probs)
+
+
+class ImageModerationNotifier(NotifierInterface):
+    """Notifier to dispatch to image moderation server
+
+    :param service_url: base url for image moderation service
+    """
+
+    def __init__(self, service_url):
+        self.service_url = service_url
+
+    def notify_image_flag(
+        self, predictions: List[Prediction], source_image: str, barcode: str
+    ):
+        """Send image to the moderation server so that a human can moderate it"""
+        image_url = settings.OFF_IMAGE_BASE_URL + source_image
+        image_id = int(source_image.rsplit("/", 1)[-1].split(".", 1)[0])
+        params = {"imgid": image_id, "url": image_url}
+        http_session.put(f"{self.service_url}/{barcode}", data=params)
+
+
+class SlackNotifier(NotifierInterface):
     """SlackNotifier implements the real SlackNotifier."""
```

Comment:
Since you are implementing NotifierInterface.

Shouldn't the comment be: ` """SlackNotifier implements the real NotifierInterface."""`

Rest looks good to me.